ts/components/services/ServicesTab.tsx

```diff
@@ -0,0 +1,262 @@
+      selectedOrganizations
+    } = this.props;
+    this.props.showModal(
+      <ChooserListContainer<Organization>
+        items={selectableOrganizations}
+        initialSelectedItemIds={some(new Set(selectedOrganizations || []))}
+        keyExtractor={(item: Organization) => item.fiscalCode}
+        itemTitleExtractor={(item: Organization) => item.name}
+        itemIconComponent={left((fiscalCode: string) =>
+          this.renderOrganizationLogo(fiscalCode)
+        )}
+        onCancel={hideModal}
+        onSave={this.onSaveAreasOfInterest}
+        isRefreshEnabled={false}
+        matchingTextPredicate={this.organizationContainsText}
+        noSearchResultsSourceIcon={require("../../../img/services/icon-no-places.png")}
+        noSearchResultsSubtitle={I18n.t("services.areasOfInterest.searchEmpty")}
+      />
+    );
+  };
+
+  private onSaveAreasOfInterest = (
+    selectedFiscalCodes: Option<Set<string>>
+  ) => {
+    if (this.props.updateOrganizationsOfInterestMetadata) {
+      this.setState({
+        toastErrorMessage: I18n.t(
+          "serviceDetail.onUpdateEnabledChannelsFailure"
+        )
+      });
+      this.props.updateOrganizationsOfInterestMetadata(selectedFiscalCodes);
+    }
+    this.props.hideModal();
+  };
+
+  private renderLocalQuickSectionDeletion = (section: ServicesSectionState) => {
+    const onPressItem = () => {
+      if (this.props.userMetadata && this.props.selectedOrganizations) {
+        const updatedAreasOfInterest = this.props.selectedOrganizations.filter(
+          item => item !== section.organizationFiscalCode
+        );
+        this.setState({
+          toastErrorMessage: I18n.t(
+            "serviceDetail.onUpdateEnabledChannelsFailure"
+          )
+        });
+        this.props.saveSelectedOrganizationItems(
+          this.props.userMetadata,
+          updatedAreasOfInterest
+        );
+      }
+    };
+    return (
+      <TouchableOpacity onPress={onPressItem}>
+        <IconFont
+          name={"io-trash"}
+          color={customVariables.brandMildGray}
+          size={17}
+          style={styles.icon}
+        />
+      </TouchableOpacity>
+    );
+  };
+
+  private onTabScroll = (tabOffset: Animated.Value) => {
+    return {
+      onScroll: Animated.event(
+        [
+          {
+            nativeEvent: {
+              contentOffset: {
+                y: tabOffset
+              }
+            }
+          }
+        ],
+        { useNativeDriver: true }
+      ),
+      scrollEventThrottle: 8 // target is 120fps
+    };
+  };
+
+  public render() {
+    return (
+      <Tab
+        activeTextStyle={styles.activeTextStyle}
+        textStyle={styles.textStyle}
+        heading={this.props.heading}
+      >
+        <ServicesSectionsList
+          isLocal={this.props.isLocal}
+          sections={this.props.sections}
+          profile={this.props.profile}
+          isRefreshing={this.props.isRefreshing}
+          onRefresh={() => this.props.onRefresh}
```

Comment:
This arrow function should be avoided. Furthermore **_this.props.onRefresh_** is called with no params (it requires a boolean parameter _hideToast_)

Reply:
updated wihtin https://github.com/teamdigitale/io-app/pull/1409/commits/1eae4b8067ba34544c82a777561d4251abf09228. 
The same commit updates also the method declaration into both the ServicesHomeScreen and the ServicesTab component

Reply:
updated wihtin https://github.com/teamdigitale/io-app/pull/1409/commits/1eae4b8067ba34544c82a777561d4251abf09228.
The same commit updates also the methods declarations into both the `ServicesHomeScreen` and the `ServicesTab` component.

Reply:
updated (https://github.com/teamdigitale/io-app/commit/1eae4b8067ba34544c82a777561d4251abf09228#diff-8f2cb07aba874cb6ee062edc768a60adR198)